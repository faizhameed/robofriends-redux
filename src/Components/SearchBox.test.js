import React from "react";
import SearchBox from "./SearchBox";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

it("expect to run SearchBox componen", () => {
  expect(shallow(<SearchBox />).length).toEqual(1);
});

it("expect to run SearchBox componen", () => {
  const wrapper = shallow(<SearchBox />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
