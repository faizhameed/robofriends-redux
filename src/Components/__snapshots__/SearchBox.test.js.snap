// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to run SearchBox componen 1`] = `
<div
  className="pa2"
>
  <input
    className="pa3 ba b--green bg-lightest-blue"
    placeholder="Search Robots"
    type="seach"
  />
</div>
`;
